persistence/src/vespa/persistence/spi/persistenceprovider.h

```diff
@@ -10,6 +10,8 @@
 #include "selection.h"
 #include "clusterstate.h"
 #include "operationcomplete.h"
+#include "bucketexecutor.h"
```

Comment:
Consider user forward declarations instead of including new header files in this header file.

Reply:
Fixed